src/api/commentType.tsx

```diff
@@ -0,0 +1,64 @@
+interface comment {
+  commentId?: number;
+  postId: number;
+  userId: string;
+  description: string;
+  postedAt: number;
+  lastUpdatedAt: number;
+}
+
+interface fetchPostCommentsApiPayload {
+  payload: {
+    postId: number;
+  };
+}
+
+interface fetchPostCommentsApiResponse {
+  message: string;
+  comments: comment[];
+}
+
+interface createPostCommentApiPayload {
+  payload: {
+    comment: comment;
+  };
+}
+
+interface createPostCommentApiResponse {
+  message: string;
+  comments: comment;
+}
+
+interface updatePostCommentApiPayload {
+  payload: {
+    comment: comment;
+  };
+}
+
+interface updatePostCommentApiResponse {
+  message: string;
```

Comment:
General comment for this file
A lot of the interfaces have the same attributes

Consider creating a base interface then extending from there using `extends`